lib/utils/utils.go

```diff
@@ -506,11 +505,6 @@ func ReadOrMakeHostUUID(dataDir string) (string, error) {
 	return id, nil
 }
 
-// PrintVersion prints human readable version
-func PrintVersion() {
```

Comment:
This change seems irrelevant to the issue.

Reply:
This was causing an import cycle because:
```
github.com/gravitational/teleport/lib/modules imports github.com/gravitational/teleport/lib/automaticupgrades imports github.com/gravitational/teleport/lib/utils imports github.com/gravitational/teleport/lib/modules: import cycle not allowed .
```

It was only used in our binaries and it was a one liner, so I just replaced the function call